packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less

```diff
@@ -31,6 +31,11 @@
         .nui-step-header-ripple {
             pointer-events: none;
         }
+
+        &:focus {
+            .focus-outline();
```

Comment:
Same comment as this one: https://github.com/solarwinds/nova/pull/332/files#r651157657